ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt

```diff
@@ -191,7 +191,7 @@ hideall "--sync--"
 982.6 "Fate Calibration α" sync / 1[56]:[^:]*:Perfect Alexander:487C:/
 985.6 "--untargetable--"
 989.8 "Ordained Motion/Stillness" sync / 1[56]:[^:]*:Perfect Alexander:49A[BC]:/
-992.8 "Obloquy, Solidarity and 3x Severity" sync  /:Perfect Alexander:4861:/
+992.8 "Obloquy, Solidarity and 3x Severity" sync / 1[56]:[^:]*:Alexander Prime:4861:/
```

Comment:
I didn't know Alexander Prime showed up to do random stuff in phase 4! That's wild.

Reply:
Fixed!

Reply:
Oh. I'm slightly disappointed that wasn't a genuine random appearance.